### PR TITLE
Woo: Clean up the woo test page in the example app

### LINKED ISSUE #1192: Woo: Split up the woo test page in the example app
The test page for Woo in the FluxC example app has become unruly. At this moment in time there are 27 buttons on this page. Too much going on. It'd be nice to have this test page instead just contain high level buttons that lead to other pages for testing specific areas of the app. We can keep the top buttons that are more specific to the app, and then add the following "group" buttons:

- Orders
- Products
- Stats

and then move the associated tests to a page accessible by that button.

### PR DESCRIPTION
**Pull request by anitaa1990**
Fixes #1192

At the moment, all the features of Woo are in a single page with over 20 buttons. This PR splits the Woo test page in the example app 3 smaller groups:
1. Products
2. Orders
3. Stats

### Testing Notes
Manually testing each button in the each of the groups to verify that the related functionality is working

### CODE REVIEW

**Comment by AmandaRiu** [comment]
@anitaa1990 Looks like this PR has some merge conflicts that need to be fixed. Once fixed and the "not ready for merge" label is removed someone will review it 👍

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -48,6 +48,31 @@
       <option name="CONTINUATION_INDENT_IN_PARAMETER_LISTS" value="false" />
       <option name="WRAP_EXPRESSION_BODY_FUNCTIONS" value="1" />
     </JetCodeStyleSettings>
+    <Objective-C-extensions>
```
Comment (AmandaRiu) [inline .idea/codeStyles/Project.xml]: @anitaa1990 Can you verify what these are added for? I don't think they are needed, but I'm not 100% sure.

Comment (aforcier) [reply .idea/codeStyles/Project.xml]: @anitaa1990 You may need to update to Android Studio 3.4 and open the project again, these should be automatically removed by that version (you'll then need to commit the removal to this branch so there are no net changes to `Project.xml`).

Some extra context: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1228

Comment (anitaa1990) [reply .idea/codeStyles/Project.xml]: @aforcier Thanks for the reference! I've been trying to ignore the file so far in all my commits and it was somewhat of a pain. Updating to 3.4 version of android studio and this is no longer a problem! Thanks 🙇‍♀ 

@AmandaRiu removed the file and if everything else is fine, then we are good to go 👍

Comment (aforcier) [reply .idea/codeStyles/Project.xml]: @anitaa1990 @AmandaRiu I think there was a slight misunderstanding 🙂 this PR removed `Project.xml` from the repo entirely, that shouldn't have happened - updating to 3.4 should have meant to net changes were committed.

Comment (anitaa1990) [reply .idea/codeStyles/Project.xml]: Oh, sorry, this is my bad! I completely misunderstood that 😕

Comment (aforcier) [reply .idea/codeStyles/Project.xml]: No worries! These things happen 🙂

**Pull request merged**